setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
opeted pr inveniosoftware/requirements-builder#6, though I'm not sure yet of the root of the issue, nor what's the correct behavior...